Common Collections/Strings/Declaring a Vector/task-info.yaml

```diff
@@ -0,0 +1,14 @@
+type: edu
+files:
+- name: src/main.rs
+  visible: true
+- name: Cargo.toml
+  visible: true
+- name: tests/tests.rs
+  visible: false
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 108
+    length: 11
+    placeholder_text: Declare your vector here
```

Comment:
Let's add //at the beginning of the placeholder text, so it won't be painted red by the Idea at the beginning of the exercise